### ISSUE TITLE
Concatenating 2 encoder outputs into one hdf5 file

### ISSUE THREAD

**Issue by SaheliDe** [keras-team/keras#10375]
Hello ,

I have two hdf5 files obtained from autoencoder (using 2 different datasets), which i need to concatenate together. But no luck concatenating :(

I tried a3 = keras.layers.concatenate([a1,a2],axis=1) but didnt work.
